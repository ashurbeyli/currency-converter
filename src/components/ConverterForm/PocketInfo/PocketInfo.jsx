import React from 'react';

const PocketInfo = ({ value, currency }) => {
    return <span>You have {value}{currency}</span>;
};

export default PocketInfo;