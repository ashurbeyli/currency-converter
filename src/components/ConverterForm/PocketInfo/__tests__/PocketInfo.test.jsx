import React from 'react';
import { mount } from 'enzyme';
import PocketInfo from '../PocketInfo';

describe('<PocketInfo />', () => {
  test('snapshot should match', () => {
    const wrapper = mount(<PocketInfo balance={100} currency="USD" />);
    expect(wrapper).toMatchSnapshot();
  });

  test('snapshot should match when balance is less than value', () => {
    const wrapper = mount(<PocketInfo value={150} balance={100} currency="USD" />);
    expect(wrapper).toMatchSnapshot();
  });
});
