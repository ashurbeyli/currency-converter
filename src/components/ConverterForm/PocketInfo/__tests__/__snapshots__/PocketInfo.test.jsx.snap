// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PocketInfo /> snapshot should match 1`] = `
<PocketInfo
  balance={100}
  currency="USD"
  value=""
>
  <span
    className={Object {}}
  >
    You have 
    100
    USD
  </span>
</PocketInfo>
`;

exports[`<PocketInfo /> snapshot should match when balance is less than value 1`] = `
<PocketInfo
  balance={100}
  currency="USD"
  value={150}
>
  <span
    className=".text-danger"
  >
    You have 
    100
    USD
  </span>
</PocketInfo>
`;
